Tests/MongoSwiftTests/AuthTests.swift

```diff
@@ -2,14 +2,15 @@ import Foundation
 @testable import MongoSwift
 import Nimble
 import TestsCommon
+import XCTest
 
 /// Represents a single file containing auth tests.
-struct AuthTestFile: Decodable {
+private struct AuthTestFile: Decodable {
```

Comment:
Updated some visibility here so that I could use the name `TestCredential` in multiple places.